web/html/src/utils/functions.test.js

```diff
@@ -0,0 +1,22 @@
+import {Utils} from "./functions";
+
+test("check order by date function'", () => {
+  const aRaw = { id: "a", date: "11/11/1983 1:10:10 CEST"};
+  const bRaw = { id: "b", date: "1983/11/11 11:11:11 CEST"};
+  const cRaw = { id: "c", date: "11/11/1983 16:16:16 CEST"};
+  let arr = [aRaw, bRaw, cRaw];
+  const sortColumnKey = "date";
+  const sortDirection = 1;
+
+  // sort ASC
+  arr = arr.sort((a, b) => Utils.sortByDate(a, b, sortColumnKey, sortDirection));
+  expect(arr[0]).toEqual(aRaw);
+  expect(arr[1]).toEqual(bRaw);
+  expect(arr[2]).toEqual(cRaw);
+
+  // sort DESC
+  arr = arr.sort((a, b) => Utils.sortByDate(a, b, sortColumnKey, -sortDirection));
+  expect(arr[2]).toEqual(aRaw);
+  expect(arr[1]).toEqual(bRaw);
+  expect(arr[0]).toEqual(cRaw);
+})
```

Comment:
Extra thanks for the tests :+1: 
Maybe add some more examples (w/o timezone, different timezones, some invalid strings)?

Reply:
Ok on second thought, since the method doesn't support comparing between different timezones and doesn't handle invalid input, these tests look sufficient.